server/mappers/headerMappers.ts

```diff
@@ -3,6 +3,9 @@ import { tabLinks } from '../data/profileBanner/profileBanner'
 import { AlertFlagLabel } from '../interfaces/alertFlagLabels'
 import { alertFlagLabels } from '../data/alertFlags/alertFlags'
 
+export const placeHolderImagePath = '/assets/images/prisoner-profile-photo.png'
+export const subStr = 'localhost:3000'
```

Comment:
This could do with being a more descriptive name  rather than `subStr` - given it's used for logic in the `pageService` and only there I think it'd be better if it was moved out of this file altogether